Controller/Adminhtml/Cart/Data.php

```diff
@@ -94,13 +104,23 @@ public function __construct(
      */
     public function execute()
     {
+        $timer = new Timer();
         try {
             $place_order_payload = $this->getRequest()->getParam('place_order_payload');
             // call the Bolt API
             $boltpayOrder = $this->cartHelper->getBoltpayOrder(true, $place_order_payload);
 
             if ($boltpayOrder) {
                 $responseData = json_decode(json_encode($boltpayOrder->getResponse()), true);
+                $latency = $timer->getElapsedTime();
+                $this->metricsClient->processCountMetric("back_office_order_token.success", 1);
+                $this->metricsClient->processLatencyMetric("back_office_order_token.latency", $latency);
+                $this->metricsClient->postMetrics();
+            } else {
+                $latency = $timer->getElapsedTime();
+                $this->metricsClient->processCountMetric("back_office_order_token.failure", 1);
+                $this->metricsClient->processLatencyMetric("back_office_order_token.latency", $latency);
+                $this->metricsClient->postMetrics();
```

Comment:
This 4 line code pattern keeps repeating. Is it reasonable to wrap it into a single method call?